lib/stellar_core_commander/transactor.rb

```diff
@@ -262,6 +262,11 @@ def clear_metrics
       @process.clear_metrics
     end
 
+    Contract None => Any
+    def protocol_version
+      @process.info_field 'protocol_version'
```

Comment:
missing `require_process_running`

Reply:
also, pattern is to add `protocol_version` instead of accessing `info_field` directly (see how `ledger_num` is implemented)

Reply:
`require_process_running` does not sound right. We want to run `protocol_version` on a specific node (hence it's called with `on :node do`). `require_process_running` will create a process if @process is null or will get whatever first process was added.

Reply:
oh I see, the name implies just running :-/
we should still add a method to process to retrieve `protocol_version` instead of using `info_field` from here

Reply:
Sounds good, updated. Also added ability to shutdown containers from transactor. Managed to test `tmpnode` shutdown with Docker locally, looks like it worked.